server/data/badger-brain.js

```diff
@@ -25,12 +25,22 @@ query {
     mailingListSummary
     featuredEvent {
       title
+      ticketsAvailable
+      waitingListOpen
       startDateTime {
         iso
       }
       endDateTime {
         iso
       }
+      ticketReleaseDate {
+        iso
+      }
+      externalLinks {
+        url
+        title
```

Comment:
I suppose the idea was to have something flexible - we might have external links that are not included in the TicketStatus, for example.